src/test/java/moduletwo10/yandex/test/TestRunner.java

```diff
@@ -0,0 +1,103 @@
+                StringСonstants.ERROR_CLICKING_TAB + MainMenu.GENERAL_ACCESS);
+        Assert.assertEquals(MainMenu.PHOTO.getTitle(), yandexDiskPage.getTabName(MainMenu.PHOTO),
+                StringСonstants.ERROR_CLICKING_TAB + MainMenu.PHOTO);
+        Assert.assertEquals(MainMenu.HISTORY.getTitle(), yandexDiskPage.getTabName(MainMenu.HISTORY),
+                StringСonstants.ERROR_CLICKING_TAB + MainMenu.HISTORY);
+        Assert.assertEquals(MainMenu.ALBUMS.getTitle(), yandexDiskPage.getTabName(MainMenu.ALBUMS),
+                StringСonstants.ERROR_CLICKING_TAB + MainMenu.ALBUMS);
+    }
+
+    @Test(description = "The test checks the work with the document, file and the recycle bin (creation, deletion)"
+            , groups = {"authorized"})
+    public void checkCreationAndDeletionOfDocumentsAndFolders() {
+        Document document = DocumentFactory.getDocument();
+        Folder folder = FolderFactory.getFolder();
+        YandexDiskPage yandexDiskPage = new YandexDiskPage();
+        SoftAssert asert = new SoftAssert();
+        YndexDiskService service = new YndexDiskService();
+        service.createNewFolder(folder.getName());
+        Boolean folderСreated = yandexDiskPage.isTheFolderContained(folder.getName());
+        asert.assertTrue(folderСreated, StringСonstants.ERROR_FOLDER_NOT_CREATED);
+
+        if (folderСreated) {
+            asert.assertEquals(yandexDiskPage
+                            .goToTab(MainMenu.FILES)
+                            .getTiteFolder(folder.getName()),
+                    folder.getName(),
+                    StringСonstants.ERROR_CLICKING_FOLDER + folder.getName());
+
+            service.createdNewWordDocument(document);
+            Boolean documentCreated = yandexDiskPage
+                    .isTheDocumentContainedInFolder(document.getName() + ".docx");
+            asert.assertTrue(documentCreated, StringСonstants.ERROR_DOCUMENT_NOT_CREATED);
+
+            if (documentCreated) {
+                asert.assertEquals(service
+                                .openDocumentAndGetTextFromDocument(document.getName() + ".docx"),
+                        document.getText(),
+                        StringСonstants.ERROR_SAVING_DATA);
+            }
+
+            service.deleteFolder(folder.getName());
+            asert.assertFalse(yandexDiskPage
+                            .isTheFolderContained(folder.getName()),
+                    StringСonstants.ERROR_FOLDER_NOT_DELETED);
+
+            asert.assertTrue(yandexDiskPage
+                            .goToTab(MainMenu.BASKET)
+                            .isTheFolderInTheBasket(folder.getName()),
+                    StringСonstants.ERROR_FOLDER_NOT_IN_BASKET);
+
+            asert.assertFalse(yandexDiskPage.clearTrash().isTheFolderInTheBasket(folder.getName()),
+                    StringСonstants.ERROR_BASKED_IS_NOT_CLEARED);
+        }
+        asert.assertAll();
+    }
+}
```

Comment:
It would be better to add hard test dependences or execution priority. And the best solution will be to make tests independent. Each test should start with a new browser opening and signing in if it needed.